crates/forge/tests/e2e/running.rs

```diff
@@ -751,84 +751,84 @@ fn should_panic() {
         output,
         indoc! { r"
         Collected 14 test(s) from should_panic_test package
+        Running 0 test(s) from src/
```

Comment:
Why this behavior changed?

Reply:
It should always be that way. I don't now why it was passing before